src/SDKs/Azure.Base/data-plane/Azure.Base/PolicyRegistration.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Base.Http.Pipeline;
+
+namespace Azure
+{
+    public struct PolicyRegistration
```

Comment:
Why in the root namespace and in general why do we need this?

Reply:
Doesn't have to be in the root namespace. We need them so TryAppend/Replace methods knew if policies are registered in some parts of the pipeline.

Reply:
Before the change the builder knew where policies are registered without the need for a new public types. I want to minimize number of public types unless such new types improve user experience, not our internal implementation.